source/MaterialXGenOgsFx/OgsFxShaderGenerator.cpp

```diff
@@ -14,49 +14,56 @@ namespace MaterialX
 namespace
 {
     // Semantics used by OgsFx
-    static const StringMap OGSFX_DEFAULT_SEMANTICS_MAP =
+    static StringMap OGSFX_DEFAULT_SEMANTICS_MAP;
```

Comment:
As this isn't really globally required so I would say just move the static map into the method:
OgsFxShaderGenerator::createShader() as that appears to be the only place that it's used.

Reply:
good point, I pushed a new version. Thanks